docs/health/index.rst

```diff
@@ -2,62 +2,137 @@
+If VRF for transfer needs to be changed from common default ones (`Mgmt-intf`, `management`), please change via `--health-mgmt-vrf` argument. for default VRF, needs to specify `None` like `iosxe:None`.
 
-pyATS Health Check comes with default checks; however it is fully open-sourced! You can add your own checks to be executed at any time! Any features or functions to monitor, those can be contributed and developed as Blitz actions or APIs.
+example when using tftp and vrf `mgmt` for both iosxe and nxos
 
-.. note::
+.. code-block:: bash
+
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-remote-device name:myserver path:/tmp/ protocol:scp --health-mgmt-vrf iosxe:mgmt nxos:mgmt
+
+**2. How to change threshold for cpu load/memory usage?**
+
+It can be done via `--health-threshold` argument.
+
+example when specifying thresholds, cpu 75% and memory 80%
+
+.. code-block:: bash
+
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-threshold cpu:75 memory:80
+
+**3. How can I change which logging keywords I want to detect?**
+
+By default, `logging` check detects only `traceback`, `Traceback` and `TRACEBACK`. The keywords can be overwritten via `--health-show-logging-keywords`.
+
+example when changing to `Crash` and `CRASH` for both iosxr and nxos.
+
+.. code-block:: bash
+
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-show-logging-keywords "iosxr:['Crash', 'CRASH']" "nxos:['Crash', 'CRASH']"
+
+**4. I want to change the location of where core files are searched for**
+
+by default, it's pre-defined for each platform (iosxe, iosxr, nxos). If you want to look somewhere else, the location can be overwritten via `--health-core-default-dir`.
+
+example to change to `harddisk0:/core` for iosxe.
+
+.. code-block:: bash
+
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-core-default-dir "iosxe:['harddisk0:/core']"
+
+**5. How to run pyATS Health check against only certain devices?**
+
+All checks will run against all connected devices in testbed yaml by default. To run checks against only specific devices, please use `--health-devices`.
+
+example to run against only `R1_xe` device.
 
-     See all the available APIs for pyATS Health Check `Available Apis <https://pubhub.devnetcloud.com/media/genie-feature-browser/docs/#/apis>`_
+.. code-block:: bash
+
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core --health-devices R1_xe
+
+**6. Is it possible to send a webex notification?**
+
+Yes. There is `--health-webex` argument to send a webex notification. The webex notification will be sent out only when health checks fail.
```

Comment:
maybe use `--health-notify-webex` so its clearer and allows for `-mail` and others

Reply:
it's already reviewed, merged and released in latest 21.6. no discussion for code change here. (we can change it later if really want, but not with this PR)